Assets/UI/launchbar.lua

```diff
@@ -32,17 +39,25 @@ local m_LaunchbarExtras:table = {};
 -- ===========================================================================
 --  Callbacks
 -- ===========================================================================
-function OnGovernmentClick()
+function OnOpenGovernment()
   local ePlayer:number = Game.GetLocalPlayer();
-  if ePlayer ~= -1 then
+  if ePlayer == -1 then
+    return; -- Probably autoplay
+  end
+
     localPlayer = Players[ePlayer];
     if localPlayer == nil then
       return;
     end
```

Comment:
this block that was in the `if ePlayer ~= -1 then` needs to be reinvented (remove 2 spaces)

Reply:
Never mind, we'll do that later.